cves/CVE-2017-5651.yml

```diff
@@ -43,15 +44,21 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
+description: |
+  This vulnerability occurs when a http request results in a send file action.
+  When a request is received resulting in a send file action, an http processor is popped from a cache. Another thread then completes the send file action
+  and pushes the http processor back into the cache. Milliseconds later, the
+  thread that popped the http processor pushes it back into the cache, causing
+  the same http processor to be in cache twice. This may lead to response mixups
+  or unexpected errors.
 
 bounty_instructions: |
   If you came across any indications that a bounty was paid out for this
   vulnerability, fill it out here. Or correct it if the information already here
   was wrong. Otherwise, leave it blank.
 bounty:
   amt:
-  announced: 2017-04-17 
+  announced:
   url:
 bugs: []
 fixes_vcc_instructions: |
```

Comment:
Was there not a VCC for this? Was it unable to be found?
The VCC below should be filled in.